client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/common/RuleDetails.java

```diff
@@ -36,4 +36,6 @@
 
   String[] getTags();
 
+  String getExtendedDescription();
```

Comment:
What is return value when there is no extended description? If it is "null" then please annotate with @CheckForNull